src/commcare_cloud/ansible/roles/nginx/vars/cchq_ssl.yml

```diff
@@ -24,7 +24,7 @@ nginx_sites:
    file_name: "{{ deploy_env }}_commcare"
    listen: "443 ssl http2{{ ' default_server' if deploy_env == primary_ssl_env else '' }}"
    root: "/var/www/html"
-   server_name: "{{ SITE_HOST }}"
+   server_name: "{{ SITE_HOST }}{% if ALTERNATE_HOSTS %} {{ ALTERNATE_HOSTS | join(' ') }}{% endif %}"
```

Comment:
I intially tried to write this as
```yml
   server_name: "{{ [SITE_HOST] + ALTERNATE_HOSTS | join(' ') }}"
```
but got an error about concatenating a list and a unicode object.  I must've been missing something obvious, but whatever, this way works.